client/src/features/session/components/status/SessionStatusText.tsx

```diff
@@ -40,13 +42,38 @@ export default function SessionStatusText({
   );
   const hibernationTimestamp =
     status === "hibernated" ? annotations["hibernation-date"] ?? "" : null;
+  const hibernationDateTime = hibernationTimestamp
+    ? ensureDateTime(hibernationTimestamp)
+    : null;
+
+  const hibernatedSecondsThreshold =
+    status === "hibernated"
+      ? parseInt(annotations?.hibernatedSecondsThreshold ?? "", 10)
+      : null;
+  const hibernationThresholdDuration =
+    !hibernatedSecondsThreshold || isNaN(hibernatedSecondsThreshold)
+      ? Duration.fromISO("")
+      : Duration.fromObject({ seconds: hibernatedSecondsThreshold });
+  const hibernationCullTimestamp =
+    hibernationDateTime && hibernationThresholdDuration
+      ? hibernationDateTime.plus(hibernationThresholdDuration)
+      : null;
 
   return status === "running" ? (
     <>Running, created {startTimeText}</>
   ) : status === "starting" ? (
     <>Starting, created {startTimeText}</>
   ) : status === "stopping" ? (
     <>Deleting...</>
+  ) : status === "hibernated" && hibernationCullTimestamp ? (
+    <>
+      Paused, will be deleted{" "}
```

Comment:
We don't need to repeat the status here. From you screenshot, the "Paused" word is already written one line above this sentence. I would go for something shorter like
```suggestion
      Deleting in{" "}
```

Reply:
"Paused" is not written in the first screenshot.

Reply:
![image](https://github.com/SwissDataScienceCenter/renku-ui/assets/43481553/1477c0e9-c1ff-4b44-a558-b5646ff30f56)

Reply:
This one: ![Screenshot 2023-08-29 at 11 22 15](https://github.com/SwissDataScienceCenter/renku-ui/assets/951086/2ce0335f-f421-48ee-adf5-43aac2ba9963)

Reply:
Ok, on the sessions tab it makes sense to keep `Paused` at the beginning.
I still believe the message on the dashboard is unnecessarily long and can be abbreviated as suggested, but I'm ok with keeping it as is